package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "plist",
   "description": "Apple's property list parser/builder for Node.js and browsers",
-  "version": "3.0.6",
+  "version": "3.0.7",
```

Comment:
don't bump the version number in this PR please, that'll be done separately.

Reply:
Reverted. Ready for another look!